Makefile

```diff
@@ -94,8 +94,13 @@ translations_pull:  ## Pull new translations from Transifex
 translations_push:  ## Push translations to Transifex
 	tox -e transifex_push
 
+COMMON_CONSTRAINTS_TXT=requirements/common_constraints.txt
+.PHONY: $(COMMON_CONSTRAINTS_TXT)
+$(COMMON_CONSTRAINTS_TXT):
+	wget -O "$(@)" https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt || touch "$(@)"
```

Comment:
@iamsobanjaved @mraarif I wanted to flag this... I found that the way this was originally implemented (`-c https://github.com/...`) breaks `dependabot` integration for security fixes; the bot/app is unable to parse the remote file, the "job" fails, and no dependabot PR is created.

By pulling down a fresh copy of the file whenever we run `make upgrade`, we can ensure we're always using the latest version, while also having a real copy available on disk, so that Github/dependabot are happy :)